pkgs/applications/graphics/qosmic/default.nix

```diff
@@ -0,0 +1,64 @@
+{ mkDerivation
+, fetchFromGitHub
+, fetchpatch
+, qmake
+, wrapQtAppsHook
+, qtbase
+, pkg-config
+, lua
+, flam3
+, libxml2
+, libpng
+, libjpeg
+, lib
+, }:
+
+mkDerivation rec {
+  pname = "qosmic";
+  version = "1.6.0";
+
+  src = #/home/aengelen/dev/qosmic;
```

Comment:
This commented out thing should not be committed.

Reply:
Oops, indeed, obviously